src/app/components/blog/components/main-dashoboard/main-dashboard.component.html

```diff
@@ -30,9 +31,14 @@ <h1 class="example-app-name">{{userLoggedIn.name}} - {{userLoggedIn.role}}</h1>
           <app-blog-page [loadFilter]="'myPosts'"></app-blog-page>
         </div>
 
-        <div *ngIf="currentState == 'approve' &&
+        <div *ngIf="currentState == 'accept' &&
+          userLoggedIn.role === 'maint'">
```

Comment:
Do we have a greater than concept here? maint or admin can accept, but would the client block the request from happening?
We could also leave the buttons and then when clicking just allow the error message to pop up saying that the user is unauthorized for that action if that's easier.